x/auth/client/cli/tx_sign.go

```diff
@@ -240,10 +241,15 @@ func makeSignCmd() func(cmd *cobra.Command, args []string) error {
 		}
 
 		overwrite, _ := f.GetBool(flagOverwrite)
+		var multisigAddr sdk.AccAddress
 		if multisig != "" {
-			multisigAddr, _, _, err := client.GetFromFields(txFactory.Keybase(), multisig, clientCtx.GenerateOnly)
+			multisigAddr, err = sdk.AccAddressFromBech32(multisig)
```

Comment:
```suggestion
			multisigAddr, err := sdk.AccAddressFromBech32(multisig)
```

we don't need the var outside.